### ISSUE TITLE
Nothing is happening when I run the example app

### ISSUE THREAD

**Issue by dheerajbred** [bostrot/flutter_applovin#1]
Nothing is happening when I run the example app. No ads are displayed

**Comment by bostrot** [comment]
Please provide a logfile or error message. Did you enter the correct AppLovin API Keys?

**Comment by dheerajbred** [comment]
No, I didn't. I found applovin SDK key but What is "YOUR_SDK_KEY_ANDROID"?
https://prnt.sc/ny11b3

**Comment by bostrot** [comment]
That is the applovin SDK key you have in your screenshot. Enter the key and report back how it works.

**Comment by mangeshdotco** [comment]
Yeah for me too!! Nothing is happening.

**Comment by bostrot** [comment]
Please post the log or the error that occurs.
